docs/architecture/adr-033-gov-split-vote.md

```diff
@@ -0,0 +1,104 @@
+# ADR 033: Governance split votes
```

Comment:
#7459 came before you, so let's reserve ADR-033 for that adr. Could you choose another number?

Reply:
@amaurymartiny will do.  Btw is there a good way to know which ADR numbers have already been claimed?